.gitignore

```diff
@@ -29,11 +29,15 @@ test/xml-results
 .idea
 cmake/projects/
 
+# Build and release products
+cmake/build/
+release/
+
 bin/api-reference-parser/vendor
 docs/api-reference
 /moai-sdk/
-/lib/

```

Comment:
I don't remember why I did this.  Can someone confirm if this should be gitignored?